app/models/group_assignment.rb

```diff
@@ -2,12 +2,16 @@ class GroupAssignment < ApplicationRecord
   include GroupAssignmentAndAssignmentScopes
 
   belongs_to :group_offer
+  validates_presence_of :group_offer
```

Comment:
Why is this [needed](https://blog.bigbinary.com/2016/02/15/rails-5-makes-belong-to-association-required-by-default.html)?

Reply:
its not, its rubbel